### PR TITLE
Fix protocol with prember

### PR DESCRIPTION
**Pull request by RobbieTheWagner**
This fixes the issue I was seeing in #1

### CODE REVIEW

File: addon/index.js [hunk 1]
```diff
@@ -15,11 +15,23 @@ export default Service.extend(AdapterMixin, {
     return owner && owner.lookup('service:fastboot');
   }),
 
+  protocol: computed(function() {
+    let protocol = get(this, 'fastboot.request.protocol');
+    // In Prember the protocol is the string 'undefined', so we default to HTTP
+    if (protocol === 'undefined:') {
+      protocol = 'http:';
+    } else if (!protocol) {
+      protocol = location.protocol;
```
Comment (tchak) [inline addon/index.js:24]: This code only runs in fastboot. How can we get here?

Comment (RobbieTheWagner) [reply addon/index.js:24]: There is no fastboot guard, is there? This runs in both the browser and fastboot, I thought? I just copied this code from @dfreeman's suggestions, and it seems to work for me.

Comment (tchak) [reply addon/index.js:24]:
```js
buildServerURL(url) {
  if (!get(this, 'fastboot.isFastBoot')) {
    return url;
  }
```

Comment (tchak) [reply addon/index.js:24]: I will merge and fix it. Thank you for researching this!

**Pull request merged**